src/controller/mapping/mapping_controller.py

```
@@ -20,13 +20,15 @@
 @version(1)
 @modes_required([TenantMode.CLASSIFIER])
 def get_saved_nomenclature_mapping_result_by_user_id(
+    iteration_key: str | None = None,
     session: Session = Depends(get_session),
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)